Corrected the grammer in the README.md file.

Puskchan opened the pull request:
Pull request title

- Clearly and concisely describes what it does
- Refer to the issue that it solves, if available


Pull request description

- Describes the main changes that come with the pull request
- Any relevant additional information is provided



> For detailed contributing instructions see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

ohbus reviewed (changes requested):
Thanks for this improvement.

Though I would like it if you could please provide a clear and concise title and description for PR.

I can understand this is your first one here. So, I would like to encourage you by starting to improve the same.

Puskchan commented:
Could you please have a look at it now? I fixed the title and added a description.

As you said correctly that this is my first PR, I Thank You for being patient and helping me out.

iluwatar commented:
Closed due to inactivity
